templates/repo/issue/view_content/delete_time.tmpl

```diff
@@ -0,0 +1,14 @@
+{{ if .comment.TimeID }} <!-- compatibility with pre v1.14 -->
```

Comment:
hmm a question in genneral if we should add a note to each template with witch version it is compatible

in general templates follow the go code ...

Reply:
`comment.Time` will be `nil` for comments created before this PR, so this check is needed..?